### PR TITLE
Improvements/544

### PR DESCRIPTION
**Pull request by RomanGalochkin**
#544

### CODE REVIEW

File: packages/server/src/lib/utils.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  });
+};
+
+/**
+ * @param className
+ * @param list
+ * @param options
+ * @param context
+ */
+module.exports.extend2 = function extend2(className, options, list, context) {
```
Comment (OAGr) [inline packages/server/src/lib/utils.js]: can there be some better name then ``extend2``, ``diff2`` and ``test2``. The 2 doesn't mean anything to me.

File: packages/server/src/lib/utils.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+/**
+ * @param className
+ * @param options
+ * @param list
+ * @return {string[]}
+ */
+module.exports.diff = function diff(className, options, list) {
+  const difference = _.difference(_.keys(options), list);
+  if (difference.length > 0) {
+    console.warn(`${className} don't expect "${difference.join(', ')}".`);
```
Comment (OAGr) [inline packages/server/src/lib/utils.js]: It should probably be ``doesn't``, not ``don't``

Comment (RomanGalochkin) [reply packages/server/src/lib/utils.js]: Yes, sorry

File: packages/server/src/lib/utils.js (hunk 1 above)
Comment (RomanGalochkin) [reply packages/server/src/lib/utils.js]: Yes, sure

**Comment by RomanGalochkin** [comment]
Updated.

**Pull request merged**